specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/examples/DdosCustomPolicyCreate.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
please move to the ```2018-11-01``` folder

Reply:
not sure there is such a folder: https://github.com/Azure/azure-rest-api-specs/tree/network-december-release/specification/network/resource-manager/Microsoft.Network/stable

what am I missing?